apex-openapi.yaml

```diff
@@ -713,26 +713,26 @@ paths:
             application/json:
               schema:
                 items:
-                  $ref: '#/components/schemas/analyzer'
```

Comment:
I can't leave a comment above because it's not a changed line: the `/_acuvity/analyzers` route we should also ignore during SDK generation. It should leave us with just one endpoint in the SDK. If we removed the helper to make it easier to get the analyzers, and we removed the possibility to pass analyzers in the helpers, then let's also remove the list function.

Reply:
I have used the list_analyzers() at many places in the code to list secrets, list piis. Let me take it another to see if there is any other route.

Reply:
ah ok, well then let's keep it if you're making use of it for these.